### PR TITLE
Drop notify settings leftovers

### PR DESCRIPTION
**Pull request by dgdavid**
## Problem

When trying to `Manage Existing Profiles`, the output of the module shows an error.

```
Can't locate Immunix/Notify.pm in @INC (you may need to install the Immunix::Notify module) (@INC contains: /usr/lib/perl5/site_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.18.2 /usr/lib/perl5/vendor_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.18.2 /usr/lib/perl5/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/5.18.2 /usr/lib/perl5/site_perl .) at /usr/lib/YaST2/servers_non_y2/ag_apparmor line 23.
BEGIN failed--compilation aborted at /usr/lib/YaST2/servers_non_y2/ag_apparmor line 23 (#1)
```

* https://bugzilla.suse.com/show_bug.cgi?id=1139852

## Why?

Because the code related to _notify settings_ [was removed some time ago](https://github.com/yast/yast-apparmor/commit/168ee1a50d496de4a7a74c2e722ca42bad8df31d), including the file [`per-lib/Notifiy.pm`](https://github.com/yast/yast-apparmor/blob/740c426f65a8132d1c1bd607306d6c3982194d4d/perl-lib/Notify.pm) which was being placed [into the `Inmmunix` perl library path](https://github.com/yast/yast-apparmor/blob/740c426f65a8132d1c1bd607306d6c3982194d4d/perl-lib/Makefile.am#L4).

However, there are a couple of `use Immunix::Notify;` in the code yet.

## Solution

To remove the leftovers related to notify settings.

---

:fire: :boom: Additionally, thanks to the [check perl syntax](https://github.com/yast/docker-yast-ruby/commit/7dd9c8cab22fa7f7d516d9664c091f19e06673be) task introduced by @lslezak few months ago in the [`yast-travis-ruby` script](https://github.com/yast/docker-yast-ruby/blob/master/yast-travis-ruby), we realized that there is another broken and not used Perl script/module/routine: `/bin/reportgen.pl`.

```
Checking yast2-apparmor.pot...
Can't locate Immunix/Reports.pm in @INC (you may need to install the Immunix::Reports module) (@INC contains: src/modules /usr/share/YaST2/modules /usr/lib/perl5/site_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.18.2 /usr/lib/perl5/vendor_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.18.2 /usr/lib/perl5/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/5.18.2 /usr/lib/perl5/site_perl .) at ./bin/reportgen.pl line 34.
BEGIN failed--compilation aborted at ./bin/reportgen.pl line 34.
``` 
<sub>^^^ output from Travis</sub>

In fact, it was deleted for SLE-15 in 025c4cc :confused: **¯\\\_(ツ)_/¯**

So, as part of the solution proposed here, all the code related to `reportgen.pl` has been deleted. :boom: :fire: 

---

## Tests

Tested by hand. The errors are gone.

### CODE REVIEW

File: package/yast2-apparmor.changes [hunk 1]
```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Jul 17 07:25:37 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Avoid crashes due to dead code related to notify settings, which
```
Comment (ancorgs) [inline package/yast2-apparmor.changes]: Crashes? I though it was only a warning in the console.

File: package/yast2-apparmor.changes [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+  were removed some time ago (bsc#1139852).
+- Removes a Perl script to generate reports that is broken
+  and is not being used from a long time ago.
```
Comment (ancorgs) [inline package/yast2-apparmor.changes]: NP: I would use "since" instead of "from"

Comment (lslezak) [reply package/yast2-apparmor.changes]: or "is not used for a long time"

**Comment by lslezak** [comment]
BTW nice to see that the perl syntax check really helps! :+1:

File: package/yast2-apparmor.changes (hunk 1 above)
Comment (dgdavid) [reply package/yast2-apparmor.changes]: Yep, for me were only _warnings_ int the console. However, in the bug report is stated 

> Error (Can't locate Immunix/Notify.pm in @INC) shows up and the "Manage Existing Profiles" is broken.

That's why I used that word. Although it worked _perfectly_ in my tests.

File: package/yast2-apparmor.changes [hunk 3]
```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Jul 17 07:25:37 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Avoid warnings due to dead code related to notify settings,
+  which were removed some time ago (bsc#1139852).
+- Removes a Perl script to generate reports that is broken
+  and is not used for a long time ago.
+- 3.2.1.1
```
Comment (dgdavid) [inline package/yast2-apparmor.changes]: Just for the record, the version number is fixed in #43 to use `3.2.2` instead.

File: package/yast2-apparmor.spec [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-apparmor
-Version:        3.2.1
+Version:        3.2.1.1
```
Comment (dgdavid) [inline package/yast2-apparmor.spec]: See the comment above, https://github.com/yast/yast-apparmor/pull/41#pullrequestreview-263046884

**Pull request merged**